docs/components/base/wheeled.md

```diff
@@ -123,7 +123,7 @@ Enter a name for your base, select the type `base`, and select the `wheeled` mod
 {{% /tab %}}
 {{< tab name="Annotated JSON" >}}
```

Comment:
```suggestion
{{% tab name="Annotated JSON" %}}
```
This image isn't rendering because the angle brackets make it not render the markdown

Reply:
Oh, did not catch that, thank you!